src/Storage/Storage.Management/Storage.Management.csproj

```diff
@@ -13,15 +13,6 @@
     <AssemblyName>$(AzAssemblyPrefix)$(PsModuleName).Management</AssemblyName>
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
-  <ItemGroup>
-    <Compile Remove="StorageAccount\AddAzureStorageAccountNetworkRule.cs" />
-    <Compile Remove="StorageAccount\GetAzureStorageAccountNetworkRuleSet.cs" />
-    <Compile Remove="StorageAccount\InvokeAzureStorageAccountFailover.cs" />
-    <Compile Remove="StorageAccount\InvokeAzureStorageAccountHierarchicalNamespaceUpgrade.cs" />
-    <Compile Remove="StorageAccount\RemoveAzureStorageAccountNetworkRule.cs" />
-    <Compile Remove="StorageAccount\RestoreAzStorageBlobRange.cs" />
-    <Compile Remove="StorageAccount\UpdateAzureStorageAccountNetworkRuleSet.cs" />
```

Comment:
We might should only add "StorageAccount\RestoreAzStorageBlobRange.cs" back, other cmdlet code still build fail.

Reply:
Done